public/index.html

```diff
@@ -0,0 +1,104 @@
+        <i class="fa fa-twitter-square"></i>
+        <i class="fa fa-youtube-square"></i>
+        </nav>
+        </nav>
+        <nav class="red-button">
+          <button><i class="fa fa-minus"></i><span>SKBT NEWS</span><i class="fa fa-minus"></i></button>
+          <button><i class="fa fa-minus"></i><span>SKATEBOARDERS</span><i class="fa fa-minus"></i></button>
+          <button><i class="fa fa-minus"></i><span>BOARD DESIGN</span><i class="fa fa-minus"></i></button>
+          <button><i class="fa fa-minus"></i><span>CONTACT US</span><i class="fa fa-minus"></i></button>
+        </nav>
+
+         <img class="top-img"src="/img/top-image.png" alt=""/>
+       <div class="icon-selector"
+         <i class="fa fa-circle-thin"></i><i class="fa fa-circle-thin"></i><i class="fa fa-circle-thin"></i>
+       </div>
+       <section class="arti">
+         <article class="content">
+           <h3>Tony Hawk Foundation Raises Big Money At Celeb Fundraiser</h3>
+           <h4>October 10, 2013</h4>
+           <p> It was just over 90 degrees in Los Angeles, but that didn't stop the throngs from coming out to celebrate the 10th annual tony Hawks's Stand up for Skateparks Benefit Presented by SLS Hotel & Casino Las Vegas on Saturday at the Green Acres Estate in Beverly Hills.</p>
+           <a href="#">[ Read More ]</a>
+         </article>
+         <article class="content">
+           <h3>Rodney Mullen: Pop an oliie and innovate! (TED Talk)</h3>
+           <h4>March 10, 2013</h4>
+           <p>The last thing Rodney Mullen, the godfather of street skating, wanted were competitive victories. In the exuberant talk he shares his love of teh open skateboarding community and how teh unique environments it plays in drive the creation of new tricks--fostering...</p>
+           <a href="#">[ Read More ]</a>
+         </article>
+         <article class="content">
+           <h3>Ray Barbee Talks 'Ban This'</h3>
+           <h4>October 04, 2013</h4>
+           <p>Since filming is in full swing for the Vans video Adventures With Chris has taken the oppertunity to sit some of the team down to look behind the skating of some of thier earlier parts. Today we sit down Ray Barbee, one of the smoothest guys to ever step on a skateboard and Chris' childhood idol...</p>
+           <a href="#">[ Read More ]</a>
+         </article>
+        </section>
+        <button class="the arrow"><span>SKATEBOARDERS</span></button>
+        <div class="img-grid">
+          <div class="img-grid__sec">
+          <h1>LANCE MOUNTAIN</h1>
+          <img class="img-small" src="img/sk8er-4.png"/>
+        </div>
+          <div class="img-grid__sec">
+          <h1>RODNEY MULLEN</h1>
+          <img class="img-small" src="img/sk8er-3.png"/>
+        </div>
+          <div class="img-grid__sec">
+          <h1>TOMMY GUERRERO</h1>
+          <img class="img-tall" src="img/sk8er-2.png"/>
+        </div>
+          <div class="img-grid__sec">
+          <h1>STEVE CABALLERO</h1>
```

Comment:
Watch out with how important you make these skater names, they are important, but `h1` tags are THE MOST IMPORTANT things on your page. The `h1` tag should be reserved for things like the brand title and should be used only once per page. `h2` is usually for something like the article title on a page and then `h3` and beyond are pretty fair game for things like this.